docs/Namespaces_in_XML_Views_2421a2c.md

```diff
@@ -11,18 +11,18 @@ view on: [help.sap.com](https://help.sap.com/viewer/DRAFT/3237636b137e43519a20ad
 
 The names of the OpenUI5 control libraries and the related subpackages are mapped to XML namespaces.
 
-One of the required namespaces can be defined as the default namespace. The control tags for this namespace do not need a prefix.
+One of the required namespaces can be defined as the default namespace \(`xmlns="..."`\). The control tags for this namespace do not need a prefix.
 
 The `<View>` tag is required and in the example below, the `mvc` namespace is defined in the first line. You can define any alias for the namespace.
 
-A control can be located in a subpackage of a control library, for example `sap.ui.layout.form.Form` is located in the `sap.ui.layout` library, but the full package name is `sap.ui.layout.form`. You have to specify this subpackage as a separate XML namespace, even if `sap.ui.layout` is already defined as namespace.
+A control can be located in a subpackage of a control library, for example `sap.ui.layout.form.Form` is located in the `sap.ui.layout` library, but the full package name is `"sap.ui.layout.form"`. You have to specify this subpackage as a separate XML namespace, even if `"sap.ui.layout"` is already defined as namespace.
```

Comment:
I usually add double quotes in formatted code if it's string literal only. But it's just my personal preference when writing something to make it more obvious that the sentence is talking about some string (`"a.b.c"`) and not an object (`a.b.c`) with that string still formatted as code (since it's part of the whole code). Is there a style guideline for contributors to reduce such "personal preferences"? Couldn't find it..